src/components/NetworkPanel/CyjsRenderer/cyjs-factory.ts

```diff
@@ -25,15 +40,28 @@ export interface CyEdge {
   }
 }
 
-const createCyNodes = (nodeViews: NodeView[], nodeSizeLocked: boolean): CyNode[] =>
+const createCyNodes = (
+  nodeViews: NodeView[],
+  nodeSizeLocked: boolean,
+): CyNode[] =>
   nodeViews.map((nv: NodeView) => {
     const data: Record<VisualPropertyName | IdType, ValueType> = {
       id: nv.id,
-      ...Object.fromEntries(nv.values.entries()),
+      ...Object.fromEntries(
+        Array.from(nv.values.entries()).map(([k, v]) => {
+          if (k === NodeVisualPropertyName.NodeShape) {
+            return [k, NodeShapeMapping[v as NodeShapeType]]
+          }
+          return [k, v]
+        }),
+      ),
```

Comment:
@d2fong Hi Dylan, I move the translate function here before giving the data to cytoscape.js.